docs/ProfilerOptions.md

```diff
@@ -0,0 +1,113 @@
+| `--clock SOURCE`   | `clock=SOURCE`    | Clock source for JFR timestamps: `tsc` (default) or `monotonic` (equivalent for `CLOCK_MONOTONIC`).                                                                                                                                                                                                                                                                                                              |
+| `--begin function` | `begin=FUNCTION`  | Automatically start profiling when the specified native function is executed.                                                                                                                                                                                                                                                                                                                                    |
+| `--end function`   | `end=FUNCTION`    | Automatically stop profiling when the specified native function is executed.                                                                                                                                                                                                                                                                                                                                     |
+| `--ttsp`           | `ttsp`            | time-to-safepoint profiling. An alias for `--begin SafepointSynchronize::begin --end RuntimeService::record_safepoint_synchronized`.</br>It is not a separate event type, but rather a constraint. Whatever event type you choose (e.g. `cpu` or `wall`), the profiler will work as usual, except that only events between the safepoint request and the start of the VM operation will be recorded.             |
+| `--libpath PATH`   | `libpath=PATH`    | Full path to libasyncProfiler.so in the container                                                                                                                                                                                                                                                                                                                                                                |
+| `--filter FILTER`  | `filter=FILTER`   | Filter threads                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
More comments on what FILTER is?

Reply:
Addressed in [6526389](https://github.com/async-profiler/async-profiler/pull/1029/commits/65263898accd8b0d26d286d63e4f1ef990ad9fbf)